components/dirigeants-section/elus-section.tsx

```diff
@@ -26,10 +26,11 @@ const ElusSection: React.FC<{ uniteLegale: IUniteLegale }> = ({
     }${(elu.nom || '').toUpperCase()}`;
 
     const infos = [
-      elu.role,
-      <>
-        {nomComplet}, né(e) en {formatDatePartial(elu.dateNaissancePartial)}
-      </>,
+      elu.role ?? <span style={{ color: '#555' }}>Non renseigné</span>,
+      <>{nomComplet}</>,
+      <span style={{ textTransform: 'capitalize' }}>
+        {formatDatePartial(elu.dateNaissancePartial)}
+      </span>,
```

Comment:
There is a js helper `capitalize()` if you want to avoid inline style (https://github.com/etalab/annuaire-entreprises-site/blob/1b8eeedee168cd46c77c0c202d56725eb459ce0d/utils/helpers/formatting/formatting.ts#L164)

Maybe that `#555` should be color placed in a class in `global.css` rather than repeated in every line ?

Also `Non renseigné` is not coloured elsewhere in the app, only italic. I think for coherence we can :
- either only use italic here
- either generalize grey to evey `Non renseigné` in the app